### ISSUE TITLE
Invalid link on the main page of the project.

### ISSUE THREAD

**Issue by NoviceF** [philsquared/Catch#447]
Hi!

The link with name [The latest, single header, version can be downloaded directly using this link] on the "https://github.com/philsquared/Catch" is lead to https://raw.githubusercontent.com/philsquared/Catch/develop/single_include/catch.hpp. Note that the develop branch, but on a master branch has newer version of Catch.

Please, edit the link.

Thanks.

**Comment by DavidEGrayson** [comment]
I started a discussion about this on the Google Group:
[Comprehensible versioning for Catch](https://groups.google.com/forum/#!topic/catch-forum/Z03uq5ujRgQ).

Since I haven't heard otherwise from @philsquared, I'm assuming that the develop branch is used to keep track of the current stable release of catch, and that if you use anything newer then you are using something that is more experimental and bleeding edge.  So we are using the develop branch in the new [MSYS2 package for catch](https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-catch/PKGBUILD).

--David

**Comment by NoviceF** [comment]
David, thanks for link, and answer. The approach of developers to versioning is not quite obvious, and this is confusing.

**Comment by philsquared** [comment]
Sorry for the silence on this. Sorting out the quirks of the versioning strategy has been on my radar for a while. Given the extra breakages and confusion recently I've picked this up properly.
I've made some changes to fully commit to semantic versioning (it had been trying to go that way for a long time), refactored my versioning script(s) to handle this more cleanly and will start tagging as well.

There is currently no develop branch, but I'll get that going again too (I wanted this change to go straight to master so we have a consistent approach).
